src/core/js/models/itemsQuestionModel.js

```diff
@@ -70,21 +70,33 @@ export default class ItemsQuestionModel extends BlendedItemsComponentQuestionMod
 
   isCorrect() {
 
+    const selectable = this.get('_selectable');
+
     const props = {
       _numberOfRequiredAnswers: 0,
+      _numberOfPartlyCorrectAnswers: 0,
       _numberOfIncorrectAnswers: 0,
       _isAtLeastOneCorrectSelection: false,
       _numberOfCorrectAnswers: 0
     };
 
     this.getChildren().each(itemModel => {
       const itemShouldBeActive = itemModel.get('_shouldBeSelected');
+      const isPartlyCorrect = itemModel.get('_isPartlyCorrect');
+
       if (itemShouldBeActive) {
         props._numberOfRequiredAnswers++;
       }
 
       if (!itemModel.get('_isActive')) return;
 
+      if (isPartlyCorrect) {
+        // force partly correct feedback and mark as correct
+        props._isAtLeastOneCorrectSelection = true;
+        props._numberOfPartlyCorrectAnswers++;
+        itemModel.set('_isCorrect', true);
+      }
+
       if (!itemShouldBeActive) {
```

Comment:
could do 
 if (itemShouldBeActive) {
   props._numberOfRequiredAnswers++;
 } else {
  props._numberOfIncorrectAnswers++;
  return;
}

Reply:
That'll not work quite as intended as the counts go out-of-wack. I'll rewrite to make it clearer.